### ISSUE TITLE
Overgang fra enumerasjoner til open type "kode" opnar for en del utfordringer

### ISSUE THREAD

**Issue by steinarhenriksen** [ks-no/fiks-arkiv-specification#101]
Overgang fra enumerasjoner til open type "kode" opnar for en del mulige utfordringer over tid.

Er me sikre på at det er ønskelig?

For eksempel blir ikkje journalposttype lenger avgrensa til eit lite utval verdier lenger, men kan ha kva som helst verdi.

I ytterste konsekvens mister det etablerte kodeverket funksjonen sin over tid.

Fordelen med å innføre `kode` er at ein kan utvide kodeverket utan å utvide skjemaet. Er det eventuelt tenkt på korleis dette litt friare kodeverket skal forvaltes "på utsida" av XML-skjemaet?

Dette er ikkje spesielt viktig sett fra eFormidlings bruk, men tenker at det kan være greit å tenke gjennom ein gong til om ein ikkje alt er ferdig med det.

Sjå f.eks. journalposttype
- KS https://github.com/ks-no/fiks-arkiv-specification/blob/main/Schema/V1/metadatakatalog.xsd#L366
- Arkivverket https://github.com/arkivverket/schemas/blob/master/N5/v3.1/metadatakatalog.xsd#L263

**Comment by sturtzel** [comment]
Ja, det er bevisst. Arkivverkets koder er beregnet på å leses av mennesker og de er beregnet på avlevering. Å ha de inne i definisjonene begrenser mulighetene for senere utvidelser. Det er f.eks. ingen kode hos Arkivverket for å arkivere en dialog (f.eks. chat i Teams). Også GeoIntegrasjon brukte kode i stedet for enum. Mange systemer bruker dessuten de korte kodene fra Noark 4 (som I, U, N, X og S). Og det finnes flere arkivsystemer som aksepterer andre koder. En revisjon av en standard eller et lovverk kan fort medføre behov for nye koder. Dette skjer f.eks. innenfor tabeller for plan- og byggesaker. Evt. begrensninger bør derfor heller ligge i oppsett.

**Comment by sturtzel** [comment]
Kodelister må dokumenteres, ja, spesielt siden det ikke finnes tjenester for å hente dem slik det var med GI.

For koder ut over standardkodene er det tre alternativer:
1: Ugyldig
2: Konfigurerbare tillegg
3: Akseptere alt

1 vs. 2/3 avhenger av Noark (sier Noark at man kan benytte egendefinerte må enten alternativ 2 eller 3 benyttes).
